src/main/java/com/maxmind/geoip2/DatabaseProvider.java

```diff
@@ -49,6 +49,29 @@ AnonymousIpResponse anonymousIp(InetAddress ipAddress) throws IOException,
     Optional<AnonymousIpResponse> tryAnonymousIp(InetAddress ipAddress) throws IOException,
             GeoIp2Exception;
 
+
+                /**
+     * Look up an IP address in a GeoIP2 IP Risk database.
+     *
+     * @param ipAddress IPv4 or IPv6 address to lookup.
+     * @return an IPRiskResponse for the requested IP address.
+     * @throws com.maxmind.geoip2.exception.GeoIp2Exception if there is an error looking up the IP
+     * @throws java.io.IOException                          if there is an IO error
+     */
+    IpRiskResponse ipRisk(InetAddress ipAddress) throws IOException,
+    GeoIp2Exception;
+
+/**
+* Look up an IP address in a GeoIP2 IP Risk database.
+*
+* @param ipAddress IPv4 or IPv6 address to lookup.
+* @return an IPRiskResponse for the requested IP address or empty if the IP address is not in the DB.
+* @throws com.maxmind.geoip2.exception.GeoIp2Exception if there is an error looking up the IP
+* @throws java.io.IOException                          if there is an IO error
+*/
+Optional<IpRiskResponse> tryIpRisk(InetAddress ipAddress) throws IOException,
+    GeoIp2Exception;
```

Comment:
This block of code is indented differently than the rest. It should have 4 spaces before it.